content/intro-to-storybook/react-native/en/get-started.md

```diff
@@ -94,6 +94,8 @@ yarn web
 
 ![3 modalities](/intro-to-storybook/app-three-modalities-rn.png)
 
+Your storybook server won't be able to load the stories list just yet, but we'll fix this in the next steps.
+
```

Comment:
@dannyhw we could probably improve this by going with:

"Checking our Storybook at this point, you might see that there's no stories displayed. That's ok, we'll take care of it shortly, for now, let's continue working on getting our application properly setup."

Something in like this

Let me know if you're ok with it

Reply:
That seems good to me, I'll update it to use your wording 👍

Reply:
its updated